native/cocos/bindings/jswrapper/v8/Object.cpp

```diff
@@ -158,6 +339,18 @@ Object *Object::createPlainObject() {
     return obj;
 }
 
+Object *Object::createMapObject() {
+    v8::Local<v8::Map> jsobj = v8::Map::New(__isolate);
+    Object *obj = _createJSObject(nullptr, jsobj);
+    return obj;
+}
+
+Object *Object::createSetObject() {
+    v8::Local<v8::Set> jsobj = v8::Set::New(__isolate);
+    Object *obj = _createJSObject(nullptr, jsobj);
+    return obj;
+}
+
```

Comment:
Do we need create weak map interface?

Reply:
V8 doesn't provide the method of creating WeakMap in c++.